app/Repositories/CircleRepository.php

```diff
@@ -98,6 +98,8 @@ public function updateCircle($circle, $request)
             'alloc_text',
             'vouching',
             'min_vouches',
+            'min_vouches_percent',
+            'calculate_vouching_percent',
```

Comment:
I think you may need to handle when circle switches the percent or or calculate percent value, it may trigger nominations to close. Or another way is to do a snapshot of the value and add it to the nominees table

Reply:
hey @Caranell, thanks for the work on this so far. does @zashton's request here make sense to you?